### PR TITLE
a11y: Add `lang` attribute to every iframe's html tag

### LINKED ISSUE #8587: html tag lacks language attribute
Every time an `<html>` tag is used, it should include a `lang` attribute.

For example:

```html
<html lang="en-US">
<html lang="de-DE">
<html lang="fr">
```

It takes the format of the 2 character language code, optionally followed by a dash and locale.

It's important for several reasons, and critical for accessibility. For a striking example of using no language code versus a language code, [watch this very short YouTube video](https://www.youtube.com/watch?v=ox5QVbZSPBk).

### PR DESCRIPTION
**Pull request by marusak**
Fixes #8587

### CODE REVIEW

**Comment by martinpitt** [comment]
Thanks for fixing this! Could this please get an assertion in check-pages?

File: pkg/shell/base_index.js [hunk 1]
```diff
@@ -33,6 +33,9 @@ function component_checksum(machine, component) {
 
 function Frames(index) {
     var self = this;
+    let language = document.cookie.replace(/(?:(?:^|.*;\s*)CockpitLang\s*=\s*([^;]*).*$)|^.*$/, "$1");
```
Comment (martinpitt) [inline pkg/shell/base_index.js]: This looks rather hairy -- isn't `cockpit.language` good enough for this? Or does it really have to be the "ll-CC" format?

File: test/verify/check-pages [hunk 2]
```diff
@@ -268,6 +268,17 @@ OnCalendar=daily
             b.key_press("echo $LANG\r")
             b.wait_in_text(line_sel(2), locale)
 
+            b.switch_to_top()
+
+            b.wait_js_func("""(function (lang) {
+                let correct = true;
+                document.querySelectorAll('#content iframe').forEach(el => {
+                    if (el.contentDocument.documentElement.lang !== lang)
+                        correct = false;
```
[the remark this answers is not in the document]
Comment (marusak) [reply test/verify/check-pages]: [NodeList](https://developer.mozilla.org/en-US/docs/Web/API/NodeList) does not have `.findIndex()`. I wrote it with `.every()` first but then find out that it is not array.

File: pkg/shell/base_index.js (hunk 1 above)
Comment (marusak) [reply pkg/shell/base_index.js]: It is the same what we use a bit lower and the same we use in the shell's `<html>`. But I could not find any reference which would say if one or another is better. So I guess `cockpit.language` would be enough. @garrett any option about it?

Comment (martinpitt) [reply pkg/shell/base_index.js]: Indeed, I wasn't aware that this is already existing code. Thanks!

File: test/verify/check-pages (hunk 2 above)
[the remark this answers is not in the document]
Comment (martinpitt) [reply test/verify/check-pages]: Thanks! Nevermind then.

**Comment by martinpitt** [comment]
This is now causing

    TypeError: Cannot read property 'documentElement' of null

**Pull request merged**